### PR TITLE
Update c2pa-node to 0.5.20 and support Node 20

### PR DESCRIPTION
**Pull request by crandmck**
I would also like to add the `uploaded_assets` dir to the repo because the script crashes if it doesn't exist.  However, because of `.gitignore` I can't add the empty dir.  Is there an easy way to do that?

### CODE REVIEW

**Comment by emensch** [comment]
I think you have two options here: 
1. modify the script to create the directory 
2. update the gitignore to something like the following:
```
uploaded_assets/*
!.gitkeep
```
basically - ignore all files _inside_ the uploaded_assets directory except for any file named `.gitkeep`. Then drop a `.gitkeep` file in the directory.

File: package.json [hunk 1]
```diff
@@ -6,14 +6,14 @@
   "main": "main/server.ts",
   "scripts": {
     "dev": "nodemon --exec 'ts-node --esm' src/server.ts",
```
Comment (emensch) [inline package.json:8]: Can you update this dev script to take advantage of your changes to the `start` script? Should be:
```
nodemon --exec 'node --loader ts-node/esm' src/server.ts"
```

Comment (crandmck) [reply package.json]: Done

**Comment by crandmck** [comment]
OK made your suggested changes.  S/be good to go.

**Review by emensch — APPROVED** [review]

**Pull request merged**